src/viz-population-over-time/VizPopulationOverTime.js

```diff
@@ -47,148 +36,159 @@ const ChartWrapper = styled(BaseChartWrapper)`
       shape-rendering: geometricPrecision;
     }
 
-    .xyframe-line {
-      transition: fill ${(props) => props.theme.transition.defaultTimeSettings};
-    }
-
     .axis.x {
       text.axis-label {
-        transform: rotate(-45deg);
+        text-anchor: end;
+        transform: translate(6px, -14px) rotate(-45deg);
       }
     }
   }
 `;
 
 const getDateLabel = (date) => format(date, "MMM d y");
 
+const BASE_MARK_PROPS = {
+  transitionDuration: {
+    fill: THEME.transition.defaultDurationMs,
+  },
+};
+
 export default function VizPopulationOverTime({
-  data: { populationOverTime },
-  dimension,
+  data,
+  defaultRangeStart,
+  setTimeRangeId,
 }) {
   const [highlighted, setHighlighted] = useState();
-
-  const dataForDimension = useMemo(() => {
-    if (!dimension) return null;
-    return populationOverTime.filter(recordIsTotalByDimension(dimension));
-  }, [dimension, populationOverTime]);
-
-  const chartData = useMemo(() => {
-    if (!dimension) return null;
-
-    return (
-      Array.from(DIMENSION_MAPPINGS.get(dimension))
-        // don't need to include unknown in this chart;
-        // they are minimal to nonexistent in historical data and make the legend confusing
-        .filter(([value]) => value !== DEMOGRAPHIC_UNKNOWN)
-        .map(([value, label]) => ({
-          label,
-          color:
-            value === TOTAL_KEY
-              ? THEME.colors.populationTimeseriesTotal
-              : THEME.colors[dimension][value],
-          coordinates: addEmptyMonthsToData({
-            dataPoints: dataForDimension.filter((record) =>
-              value === TOTAL_KEY
-                ? true
-                : record[DIMENSION_DATA_KEYS[dimension]] === value
-            ),
-            monthCount: EXPECTED_MONTHS,
-            valueKey: "population_count",
-            emptyValue: "0",
-            dateField: "population_date",
-          })
-            .map((record) => ({
-              time: parseISO(record.population_date),
-              population: Number(record.population_count),
-            }))
-            .sort((a, b) => ascending(a.time, b.time)),
-        }))
-    );
-  }, [dataForDimension, dimension]);
-
-  if (!dimension) return null;
+  const [dateRangeStart, setDateRangeStart] = useState();
+  const [dateRangeEnd, setDateRangeEnd] = useState();
+  const { isMobile } = useBreakpoint();
+
+  useEffect(() => {
+    if (defaultRangeStart) {
```

Comment:
It may be worth validating that `defaultRangeStart` is strictly before the value for `dateRangeEnd` just below to ensure we don't have non-sensical results in some future code-drifted world.